src/index.tsx

```diff
@@ -1,5 +1,6 @@
 import ReactDOM from 'react-dom';
 import 'bulma/css/bulma.css';
+// where is it file came from (bulma)?
```

Comment:
Don't leave comments like that. Answering your question% it is a CSS framework

Reply:
I've actually forgotten  about this question)) And missed it while preparing code to commit)